pkg/ccl/changefeedccl/kvevent/event.go

```diff
@@ -100,40 +100,40 @@ type Event struct {
 }
 
 // Type returns the event's Type.
-func (b *Event) Type() Type {
-	if b.kv.Key != nil {
+func (e *Event) Type() Type {
```

Comment:
I would revert this change and do this minor refactoring in separate PR.
(not that I mind it, it's just such refactors make subsequent backports more painful).